zoom_sesh.py

```diff
@@ -259,12 +285,28 @@ def _group_split(self, by, arg, group_size):
 
     return extras, prev_combos
 
+  # Output and display funcs
+  # ====================================================
+  def save_breakout(self, i):
```

Comment:
Is the i argument in the save_breakout function so that the user could choose to save a specific breakout manually? For example: "That 3rd breakout was interesting, I shall call zoomSesh.save_breakout(2)"?

Reply:
That was the original intent, but It sounds a bit goofy. Right now, this function being called internally inside of `breakout`:

https://github.com/nneibaue/alumni_shuffler/blob/09a1b8c09f74c38ef21a7f580d7735178e824a22/zoom_sesh.py#L153-L155

Moving forward, I don't think saving breakouts should be optional... we should always save them. What do you think?

Reply:
In staging, I am working on methods to save and retrieve anonymous breakout data so they can send us the results of a session without violating FERPA. Lynde texted me and said there were a few overlaps in the most recent session, but they can't send us the breakout data without having to manually go anonymize it.

See here: https://github.com/nneibaue/alumni_shuffler/commit/feabee2cfb3772f3ac4b975e30e959838627d835